vite.config.js

```diff
@@ -24,7 +24,13 @@ export default defineConfig(function ({ mode }) {
        * Issue: https://github.com/vitejs/vite/issues/2415.
        */
       pluginRewriteAll(),
-      vue(),
+      vue({
+        template: {
+          compilerOptions: {
+            whitespace: 'preserve',
+          },
+        },
+      }),
```

Comment:
No. It’s for any view where you render a template like this:

```html
<p>
  one
  two
  three
</p>
```

With the default setting of Vue CLI/Vite (`'condense'`), the resulting HTML looks like this:

```html
<p>onetwothree</p>
```

Which is completely in contrast with how HTML normally renders which would be (unsurprisingly):

```html
<p>
  one
  two
  three
</p>
```

This is quite important to me because the browser displays both things differently. Since the spaces between the words are removed in one example, the browser would actually show “onetwothree” while in the other example it would be “one two three”.

I don’t know why the Vue folks decided on this non-standard whitespace processing default for templates, but I strongly dislike it.

Reply:
Oh wow yeah thats unfortunate 😄 . Could you point me to where that was an issue in this PR, surprised we've got this far without it being an issue, oh mind you I suppose its since we've moved to vite maybe?

Reply:
Since moving to tables, this didn’t happen in this PR, but I’ve worked around this in other instances by turning things into horizontal lists. Don’t remember where though.